### PR TITLE
Add multiple canvas sample.

### PR DESCRIPTION
**Pull request by greggman**
Note: AFAICT the stanford dragon was getting flat shading. If that was the intent I can remove the genarateNormals code but I assumed the intent was smooth shading.

I do kind of wonder if we should just genearte then once and put them in the data file 😛

### CODE REVIEW

**Review by shrekshao — APPROVED** [review]
New sample looks great!

**Comment by shrekshao** [comment]
> Note: AFAICT the stanford dragon was getting flat shading. If that was the intent I can remove the genarateNormals code but I assumed the intent was smooth shading.

No I don't think flat shading is specially intended.

> I do kind of wonder if we should just genearte then once and put them in the data file 😛

I'm good with either way. In that case less effort is prferred, so we can just keep things as-is :)

File: meshes/utils.ts [hunk 1]
```diff
@@ -33,6 +33,170 @@ export function computeSurfaceNormals(
...
+    );
+  }
+
+  let tempVerts = {};
+  let tempVertNdx = 0;
+
+  // this assumes vertex positions are an exact match
+
+  function getVertIndex(vert: [number, number, number]): number {
+    const vertId = `${vert[0]},${vert[1]},${vert[2]}`;
```
[the remark this answers is not in the document]
Comment (greggman) [reply meshes/utils.ts]: changed to `JSON.stringify(vert)` which works.

**Comment by kainino0x** [comment]
There is #302 about artifacts in the dragon model because I think we are using the original 3d scanned model file instead of a cleaned-up version. If we replace it then we can add normals and stuff.

**Pull request merged**